tox.ini

```diff
@@ -15,6 +15,13 @@ deps =
     flake8==2.4.0
 commands=flake8 redshift_sqlalchemy tests
 
+[testenv:docs]
+changedir=docs
+deps=
+    -rrequirements-docs.txt
+commands=
+    sphinx-build -W -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
+
 [pytest]
-addopts = --doctest-modules --doctest-glob='*.rst' --ignore=setup.py
+addopts = --doctest-modules --doctest-glob='*.rst' --ignore=setup.py --ignore=docs/conf.py
```

Comment:
I believe the relieves you from having to explicitly ignore all these non-test Python files.